src/+otp/+ascherlineardae/AscherLinearDAEProblem.m

```diff
@@ -21,7 +21,8 @@ function onSettingsChanged(obj)
         function y = internalSolveExactly(obj, t)
             beta = obj.Parameters.Beta;
             if ~isequal(obj.Y0, [1; beta])
-                error('An exact solution is unavailable for this initial condition');
+                error('OTP:noExactSolution', ...
```

Comment:
It seems like there is no consistent casing, and lower case 'no' has precedent in matlab. Weird, but this seems correct.